frontend/src/scenes/dashboard/dashboardLogic.ts

```diff
@@ -27,6 +26,7 @@ import { teamLogic } from '../teamLogic'
 import { urls } from 'scenes/urls'
 import { userLogic } from 'scenes/userLogic'
 import { mergeWithDashboardTile } from 'scenes/insights/utils/dashboardTiles'
+import { dayjs, now } from 'lib/dayjs'
```

Comment:
I think these were forgotten to be removed?

Reply:
Interestingly. We use dayjs elsewhere in the file already but we never return the `Dayjs` type. So kea typegen never has to deal with it. 

I thought I'd not rewrite existing functionality since there's a good chance we leave this in for a few days to get a snapshot of data and then remove it